.travis.yml

```diff
@@ -7,6 +7,8 @@ cache:
   bundler: true
   directories:
     - node_modules
+before_cache:
+  - find node_modules -type f -name "*.node" -delete
```

Comment:
this is a work-around for a few .node files built in the canvas package

```
node_modules/canvas/build/Release/obj.target/canvas.node
node_modules/canvas/build/Release/canvas.node
```

Reply:
I'm not 100% sure this is correct. Or - if only these two files (and not canvas) are removed, will canvas be able to run?

Reply:
Is this something we only need on Travis? Should we also (temporarily) add this to `bin/setup` before the call to `yarn` on line 26?

Reply:
@djuber After I cleaned out the `canvas.node` files I had some trouble reinstalling `node-canvas` on my M1 Mac. I did some digging and found the list of necessary dependencies in their issue tracker:

```
brew install pkg-config cairo pango libpng jpeg giflib librsvg
```

The same issue also pointed at making sure that the include path for C++ is set correctly:

```
 CPLUS_INCLUDE_PATH=/opt/homebrew/include yarn install
```

This wasn't an issue for me, as I already set `$CPPFLAGS` to `-I /opt/homebrew/include` in my shell config. 

Anyway, just figured I'll add this information here so we have it documented if other team members run into this.

Reply:
eacd013d285dd68fe1e889301d913a9ac47b818c adds a cleanup step to bin/setup (removes the canvas module, and reinstalls it).

That's unfortunately a little wasteful, but I can't determine the right way to check if the installed extension is compatible with the running node version (and rebuild can be skipped) - so we'll just have to follow up to remove this afterward.